2020-05-22T03:52:54.262Z error: Error in getting cp by email [ 'amit.arora15@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:13:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T03:54:29.551Z error: Error in getting cp by email [ 'amit.arora15@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:13:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T03:55:32.142Z error: Error in getting cp by email [ 'amit.arora15@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:13:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T04:15:37.631Z error: Error in getting cp by email [ 'amit.arora15@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:15:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T04:16:27.638Z error: Error in getting cp by email [ 'yash.raj@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:15:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T04:16:57.783Z error: Error in getting cp by email [ 'yash.raj@gmail.com' ] TypeError: Cannot read property '0' of undefined
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:15:56)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-22T04:22:14.785Z error: Error in getting cp by email [ 'yash.raj@gmail.com' ] Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''yash.raj@gmail.com')' at line 3
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpsByEmail [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''yash.raj@gmail.com')' at line 3",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    "                                                      where email = ANY('yash.raj@gmail.com')"
}
2020-05-24T18:38:21.538Z error: Error in getting movies by title %P% - TypeError: collection.map is not a function
    at module.exports (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/utils/orderCollectionByKey.js:8:23)
    at DataLoader.getMoviesByTitle [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/movie.js:28:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-24T18:42:21.940Z error: Error in getting movies by title %P% - TypeError: collection.map is not a function
    at module.exports (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/utils/orderCollectionByKey.js:8:23)
    at Object.getMoviesByTitle (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/movie.js:28:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 2)
2020-05-24T18:42:38.978Z error: Error in getting cp by ids [ 1, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(1, 2)'
}
2020-05-24T19:10:02.166Z error: Error in getting shows by cpids [ 2 ] - Error: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (events.js:421:28)
    at Socket.emit (events.js:315:20)
    at Socket._onTimeout (net.js:480:8)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
2020-05-24T19:10:13.584Z error: Error in getting movies by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.releaseConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:157:10)
    at Pool._removeConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:277:8)
    at Pool._purgeConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:258:8)
    at Ping.onOperationComplete (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:101:12)
    at Ping.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Ping._callback (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:488:16) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T19:12:06.351Z error: Error in getting shows by cpids [ 2 ] - Error: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (events.js:421:28)
    at Socket.emit (events.js:315:20)
    at Socket._onTimeout (net.js:480:8)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
2020-05-24T19:12:46.663Z error: Error in getting movies by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.releaseConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:157:10)
    at Pool._removeConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:277:8)
    at Pool._purgeConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:258:8)
    at Ping.onOperationComplete (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:101:12)
    at Ping.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Ping._callback (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:488:16) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T19:16:06.812Z error: Error in getting shows by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T19:19:59.918Z error: Error in getting shows by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T19:20:32.940Z error: Error in getting shows by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T19:41:52.873Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '$1))
                 UNION  
                 (select ingested_by, ingested_on,' at line 1
2020-05-24T19:42:24.012Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '$1))
                 UNION  
                 (select ingested_by, ingested_on,' at line 1
2020-05-24T19:43:56.629Z error: Error : ER_BAD_FIELD_ERROR: Unknown column '$1' in 'where clause'
2020-05-24T19:44:23.611Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?))' at line 2
2020-05-24T19:45:53.424Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?)' at line 2
2020-05-24T19:47:01.254Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?)' at line 2
2020-05-24T19:48:32.279Z error: Error : ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?)' at line 2
2020-05-24T19:50:23.002Z error: Error : argument callback must be a function when provided
2020-05-24T19:50:40.617Z error: Error : argument callback must be a function when provided
2020-05-24T19:51:36.486Z error: Error : argument callback must be a function when provided
2020-05-24T19:51:49.639Z error: Error in getting shows by cpids [ 2 ] - Error: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (events.js:421:28)
    at Socket.emit (events.js:315:20)
    at Socket._onTimeout (net.js:480:8)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
2020-05-24T20:09:03.888Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:11:52.585Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:12:50.589Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:17:39.916Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:19:23.064Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:19:49.517Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:29:01.163Z error: Error in getting shows by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T20:29:01.171Z error: Error : Handshake inactivity timeout
2020-05-24T20:29:53.749Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:31:08.288Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:31:59.808Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:33:15.246Z error: Error in getting shows by cpids [ 2 ] - Error: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:315:20)
    at Handshake._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:48:16)
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:202:8)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getShowsByCpIds [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/show.js:12:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29) {
  code: 'PROTOCOL_SEQUENCE_TIMEOUT',
  fatal: true,
  timeout: 10000
}
2020-05-24T20:33:15.255Z error: Error : Handshake inactivity timeout
2020-05-24T20:34:35.912Z error: Error in getting cp by ids [ 2, 2 ] - Error: ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT: Incorrect parameter count in the call to native function 'ANY_VALUE'
    at Query.Sequence._packetToError (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    --------------------
    at Pool.query (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mysql/lib/Pool.js:199:23)
    at internal/util.js:297:30
    at new Promise (<anonymous>)
    at Pool.<anonymous> (internal/util.js:296:12)
    at DataLoader.getCpById [as _batchLoadFn] (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:24:38)
    at dispatchBatch (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:275:29)
    at /Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/dataloader/index.js:258:12
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 'ER_WRONG_PARAMCOUNT_TO_NATIVE_FCT',
  errno: 1582,
  sqlMessage: "Incorrect parameter count in the call to native function 'ANY_VALUE'",
  sqlState: '42000',
  index: 0,
  sql: 'select *\n' +
    '                                                      from content_partner.content_partner\n' +
    '                                                      where cp_id = ANY_VALUE(2, 2)'
}
2020-05-24T20:35:58.535Z error: Error : read ECONNRESET
2020-05-24T21:08:15.083Z error: Error in getting cp by ids 4 - TypeError: collection.map is not a function
    at module.exports (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/utils/orderCollectionByKey.js:8:23)
    at Object.getCpById (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:27:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-24T21:09:57.580Z error: Error in getting cp by ids 6 - TypeError: collection.map is not a function
    at module.exports (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/utils/orderCollectionByKey.js:8:23)
    at Object.getCpById (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/lib/dao/mysql/cp.js:27:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-05-24T22:18:52.494Z error: Error in getting count for cpIds 12 - MongoError: $in needs an array
    at Connection.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/pool.js:450:61)
    at Connection.emit (events.js:315:20)
    at processMessage (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    at readableAddChunk (_stream_readable.js:276:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  ok: 0,
  errmsg: '$in needs an array',
  code: 2,
  codeName: 'BadValue',
  [Symbol(mongoErrorContextSymbol)]: {}
}
2020-05-24T22:30:15.080Z error: Error in getting count for cpIds 12 - MongoError: $in needs an array
    at Connection.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/pool.js:450:61)
    at Connection.emit (events.js:315:20)
    at processMessage (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    at readableAddChunk (_stream_readable.js:276:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  ok: 0,
  errmsg: '$in needs an array',
  code: 2,
  codeName: 'BadValue',
  [Symbol(mongoErrorContextSymbol)]: {}
}
2020-05-24T22:31:26.270Z error: Error in getting count for cpIds 12 - MongoError: $in needs an array
    at Connection.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/pool.js:450:61)
    at Connection.emit (events.js:315:20)
    at processMessage (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    at readableAddChunk (_stream_readable.js:276:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  ok: 0,
  errmsg: '$in needs an array',
  code: 2,
  codeName: 'BadValue',
  [Symbol(mongoErrorContextSymbol)]: {}
}
2020-05-24T22:32:16.947Z error: Error in getting count for cpIds 12 - MongoError: $in needs an array
    at Connection.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/pool.js:450:61)
    at Connection.emit (events.js:315:20)
    at processMessage (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    at readableAddChunk (_stream_readable.js:276:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  ok: 0,
  errmsg: '$in needs an array',
  code: 2,
  codeName: 'BadValue',
  [Symbol(mongoErrorContextSymbol)]: {}
}
2020-05-24T22:33:52.397Z error: Error in getting count for cpIds 12 - Error: $in needs an array
    at Connection.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/pool.js:450:61)
    at Connection.emit (events.js:315:20)
    at processMessage (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/amitarora/Amit/workspace/poc-workspace/nodejs/node-graphql-mysql-mongo/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:300:12)
    at readableAddChunk (_stream_readable.js:276:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  ok: 0,
  errmsg: '$in needs an array',
  code: 2,
  codeName: 'BadValue',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}
